tower-grpc-examples/build.rs

```diff
@@ -6,12 +6,19 @@ fn main() {
         .enable_server(true)
         .enable_client(true)
         .build(&["proto/helloworld/helloworld.proto"], &["proto/helloworld"])
-        .unwrap();
+        .unwrap_or_else(|e| panic!("protobuf compilation failed: {}", e));
 
     // Build routeguide
     tower_grpc_build::Config::new()
         .enable_server(true)
         .enable_client(true)
         .build(&["proto/routeguide/route_guide.proto"], &["proto/routeguide"])
-        .unwrap();
-}
+        .unwrap_or_else(|e| panic!("protobuf compilation failed: {}", e));
+
+    // Build grpc-interop
+    tower_grpc_build::Config::new()
```

Comment:
Should this be here or in the `tower-grpc-interop` crate?

I don't really have much of an opinion either way, it was initially unexpected to me.

Reply:
Whoops, I meant to remove that --- I think I forgot to commit it. `tower-grpc-interop` *should* have its own `build.rs` unless I also forgot to commit _that_...